mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -345,10 +390,12 @@ CLASS aij net/minecraft/entity/Entity
 	METHOD bQ getServer ()Lnet/minecraft/server/MinecraftServer;
 	METHOD bR isImmuneToExplosion ()Z
 	METHOD bS entityDataRequiresOperator ()Z
+	METHOD bT teleportRequested ()Z
 	METHOD bU getPrimaryPassenger ()Laij;
 	METHOD bV getPassengerList ()Ljava/util/List;
 	METHOD bW getPassengersDeep ()Ljava/util/Collection;
-	METHOD bY getTopmostVehicle ()Laij;
+	METHOD bX hasPlayerJockey ()Z
+	METHOD bY getPrimaryVehicle ()Laij;
```

Comment:
What's the reasoning for not just calling it `getBottommostVehicle` ?

Reply:
Trying to avoid MCP names, and in this case the bottom-most vehicle is also the vehicle being steered, the primary means of transportation, so I felt Primary Vehicle would be more suitable.

Reply:
`getRootVehicle`

Reply:
Props to @liach , the MVP of method names.